tests/support/qunit_api.ts

```diff
@@ -5,12 +5,21 @@
 /* eslint-disable */
 
 // global.VF == Vex.Flow and is available in all of the tests.
+// TODO: Remove this after we have cleaned up the VF.* prefixes in the tests.
 declare global {
   var VF: any;
 }
-declare const global: any;
 
-// Declarations to interface with ./support/qunit.js
+// Declarations to interface with ./qunit.js
+//
+// We are currently using QUnit 1.19.0.
+//
+// According to https://qunitjs.com/upgrade-guide-2.x/
+// QUnit 2 no longer uses global functions. The assertion methods
+// are exposed through an assert object that is bound to each test.
+//
+// However, we stick with the global methods for readability.
+declare const global: any;
 export const QUnit = global.QUnit;
 export const test = global.test;
 export const expect = global.expect;
```

Comment:
Is this still required? These are no longer imported from `declarations`?

Reply:
I actually just renamed `declarations.ts` and moved it into the `support/` folder. I realized that declarations was all about QUnit types. It's good to keep this around because otherwise TypeScript complains about the QUnit globals in all the test files.

Reply:
your choice :)